implants/lib/eldritch/src/process/kill_impl.rs

```diff
@@ -20,49 +22,48 @@ pub fn kill(pid: i32) -> Result<()> {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use std::{process::Command};
+    use std::process::Command;
 
     #[test]
-    fn test_process_kill() -> anyhow::Result<()>{
+    fn test_process_kill() -> anyhow::Result<()> {
         let mut commandstring = "sleep 5";
-        if cfg!(target_os = "linux") || 
-                cfg!(target_os = "ios") || 
-                cfg!(target_os = "macos") || 
-                cfg!(target_os = "android") || 
-                cfg!(target_os = "freebsd") || 
-                cfg!(target_os = "openbsd") ||
-                cfg!(target_os = "netbsd") {
+        if cfg!(target_os = "linux")
+            || cfg!(target_os = "ios")
+            || cfg!(target_os = "macos")
+            || cfg!(target_os = "android")
+            || cfg!(target_os = "freebsd")
+            || cfg!(target_os = "openbsd")
+            || cfg!(target_os = "netbsd")
+        {
             commandstring = "sleep";
         } else if cfg!(target_os = "windows") {
             commandstring = "timeout";
         }
-        
-        let child = Command::new(commandstring)
-            .arg("120")
-            .spawn()?;
+
+        let child = Command::new(commandstring).arg("120").spawn()?;
 
         let mut sys = System::new();
-        sys.refresh_processes();    
-        for (pid, _) in sys.processes() {
-            if pid.as_u32() == child.id(){
+        sys.refresh_processes();
+        for pid in sys.processes().keys() {
```

Comment:
Wow that’s cool that it can update that 😎

Reply:
It didn't do it itself, but it did give me the code to just handle it.

Clippy can auto-fix a lot with `--fix`, but I also did a lot of manual changes based on its asks.